src/Cohesity.Powershell/Scripts/View/Add-CohesityViewWhitelist.ps1

```diff
@@ -0,0 +1,91 @@
+function Add-CohesityViewWhitelist {
+    <#
+        .SYNOPSIS
+        Add whitelist IP(s) for a given view.
+        .DESCRIPTION
+        Add whitelist IP(s) for a given view.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Add-CohesityViewWhitelist -IP4List "1.1.1.1", "2.2.2.2" -NetmaskIP4 "255.255.255.0"
+        Add whitelist IP(s) an override global whitelist for a given view.
+        .EXAMPLE
+        Add-CohesityViewWhitelist -IP4List "1.1.1.1", "2.2.2.2" -NetmaskIP4 "255.255.255.0" -NFSRootSquash -NFSAccess "kReadWrite" -NFSAllSquash -SMBAccess "kReadWrite"
```

Comment:
Of late got another request: 

```
I see the Powershel Properties , Its taking the SID not the User ID
```

Can we solve setting ACL / whitelist with a single cmdlet or setter ?

Reply:
We can have a single setter for whitelist. Did not get the context for "Its taking the SID not the User ID".